services/venice-server/src/test/java/com/linkedin/venice/listener/ErrorCountAppender.java

```diff
@@ -57,7 +57,9 @@ public ErrorCountAppender build() {
 
   @Override
   public void append(LogEvent event) {
-    if (event.getLevel().equals(Level.ERROR) && event.getThrown().getLocalizedMessage().contains(exceptionMessage)) {
+    if (event.getLevel().equals(Level.ERROR) && event.getThrown() != null
```

Comment:
Could you mention this change also in the commit message?

Reply:
Updated.